python/pyarrow/tests/test_orc.py

```diff
@@ -128,42 +129,62 @@ def test_example_using_json(filename, datadir):
     """
     # Read JSON file
     path = datadir / filename
-    table = pd.read_json(str(path.with_suffix('.jsn.gz')), lines=True)
+    table = pd.read_json(str(path.with_suffix(".jsn.gz")), lines=True)
     check_example_file(path, table, need_fix=True)
 
 
 def test_orcfile_empty(datadir):
     from pyarrow import orc
 
-    table = orc.ORCFile(datadir / 'TestOrcFile.emptyFile.orc').read()
+    table = orc.ORCFile(datadir / "TestOrcFile.emptyFile.orc").read()
     assert table.num_rows == 0
 
-    expected_schema = pa.schema([
-        ('boolean1', pa.bool_()),
-        ('byte1', pa.int8()),
-        ('short1', pa.int16()),
-        ('int1', pa.int32()),
-        ('long1', pa.int64()),
-        ('float1', pa.float32()),
-        ('double1', pa.float64()),
-        ('bytes1', pa.binary()),
-        ('string1', pa.string()),
-        ('middle', pa.struct([
-            ('list', pa.list_(pa.struct([
-                ('int1', pa.int32()),
-                ('string1', pa.string()),
-                ]))),
-            ])),
-        ('list', pa.list_(pa.struct([
-            ('int1', pa.int32()),
-            ('string1', pa.string()),
-            ]))),
-        ('map', pa.list_(pa.struct([
-            ('key', pa.string()),
-            ('value', pa.struct([
-                ('int1', pa.int32()),
-                ('string1', pa.string()),
-                ])),
-            ]))),
-        ])
+    expected_schema = pa.schema(
+        [
+            ("boolean1", pa.bool_()),
+            ("byte1", pa.int8()),
+            ("short1", pa.int16()),
+            ("int1", pa.int32()),
+            ("long1", pa.int64()),
+            ("float1", pa.float32()),
+            ("double1", pa.float64()),
+            ("bytes1", pa.binary()),
+            ("string1", pa.string()),
+            (
+                "middle",
+                pa.struct(
+                    [
+                        (
+                            "list",
+                            pa.list_(
+                                pa.struct(
+                                    [("int1", pa.int32()), ("string1", pa.string())]
+                                )
+                            ),
+                        )
+                    ]
+                ),
+            ),
+            (
+                "list",
+                pa.list_(pa.struct([("int1", pa.int32()), ("string1", pa.string())])),
+            ),
+            (
+                "map",
+                pa.list_(
+                    pa.struct(
+                        [
+                            ("key", pa.string()),
+                            (
+                                "value",
+                                pa.struct(
+                                    [("int1", pa.int32()), ("string1", pa.string())]
+                                ),
+                            ),
+                        ]
+                    )
+                ),
+            ),
+        ]
+    )
```

Comment:
This is a typical place where auto-formatting isn't doing well at all, IMHO.

Reply:
Yes, I agree this is horrible. But in this case it could be rather easily "solved" by defining those 3 types on itself just before creating the schema. 

Although it would still not be beautiful. The most nested one from above would become:

```
df test():
    ...
    map_type = pa.list_(
        pa.struct(
            [
                ("key", pa.string()),
                ("value", pa.struct([("int1", pa.int32()), ("string1", pa.string())])),
            ]
        )
    )
    expected_schema = ...
```